app/texters/member_texter.rb

```diff
@@ -0,0 +1,32 @@
+class MemberTexter < BaseTexter
+  include ActionView::Helpers::TextHelper
+
+  attr_reader :member
+
+  def initialize(member)
+    @member = member
+  end
+
+  def overdue_notice(summaries)
+    return unless member.reminders_via_text?
+
+    message = <<~EOM
+      Chicago Tool Library Reminder: You have #{pluralize(summaries.length, "overdue item")}! Please schedule a return appointment at #{account_member_url}
```

Comment:
I think that `summaries` can contain items that are not overdue but merely checked out. You can use `LoanSummary#overdue_as_of` to determine if the summary is for an overdue item.

This worked for the emails because they render the entire state of your account, and the items that are overdue [get called out](https://github.com/chicago-tool-library/circulate/blob/main/app/views/member_mailer/_summaries.erb#L13-L17).

Reply:
Good call to double check! It looks like these summaries are already [filtered to be overdue only](https://github.com/chicago-tool-library/circulate/blob/sms-reminders/app/lib/activity_notifier.rb#L18) before being passed in. You can see the [overdue template](https://github.com/chicago-tool-library/circulate/blob/a3db478b5cb2ac7c1893c3cf18946852d55ebaa6/app/views/member_mailer/_overdue.erb) just displays all of them them without filtering.